introduction.md

```diff
@@ -1,2 +1,9 @@
 Hi I'm Łukasz, and I want to master github.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?